PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+<!-- Describe the future self what has changed and most importanly why -->
+**What changed? Why?**
+
+
+<!-- How have you verified this change? Tested locally? Added a unit test? Checked in staging env? -->
+**How has it been tested?**
+
+
+<!-- How risky is it to deploy this straight to production? -->
+**Potential risk level (on a scale from 1 to 5)**
```

Comment:
Could it be more clear about the scale? 1 is the highest risk or 5 is the highest risk?

Reply:
In addition, the number could be arbitrary if there is no criteria about risk level. What about changing the question to: What could be broken (What is the worst case) when deploying this change to production?

Reply:
I'll clarify this, thanks for pointing out!